using/submodules

```diff
@@ -0,0 +1,48 @@
+# Use Git submodules
+
+## Clone submodules during deployment
+
+Platform.sh allows you to use submodules in your Git repository. They are usually
+listed in a ``.gitmodules`` file at the root of your Git repository. When you push via Git, 
```

Comment:
There's no need for double ` in markdown, a single one as in:

    `.gitmodules`

is fine